tests/limit3/constraints/test.hal

```diff
@@ -66,47 +66,6 @@ net out-vel => ddt.out-accel.in
 net out-accel <= ddt.out-accel.out
 
 
-#
-# These comparators output True when the limit3 is obeying its
-# constraints.
-#
-
-loadrt wcomp names=wcomp.accel,wcomp.vel,wcomp.pos
-
-# Just a hair above limit3's accel limit.
-setp wcomp.accel.max 15801
-setp wcomp.accel.min -15801
-net out-accel => wcomp.accel.in
-net accel-ok <= wcomp.accel.out
-
-# Just a hair above limit3's velocity limit.
-setp wcomp.vel.max 1501
-setp wcomp.vel.min -1501
-net out-vel => wcomp.vel.in
-net vel-ok <= wcomp.vel.out
-
-setp wcomp.pos.max 105
-setp wcomp.pos.min -105
-net output => wcomp.pos.in
-net pos-ok <= wcomp.pos.out
-
-
-#
-# These AND components combine the three constraints from above and
-# compute a single "all constraints are ok" signal.
-#
-
-loadrt and2 count=2
-net vel-ok => and2.0.in0
-net accel-ok => and2.0.in1
-
-net derivatives-ok <= and2.0.out
-net derivatives-ok => and2.1.in0
-net pos-ok => and2.1.in1
-
-net constraints-ok <= and2.1.out
-
-
```

Comment:
This was all leftovers from when i was testing it by hand using halscope, this cleanup is welcome.